src/emails/templates/monthlyActivityFree/MonthlyActivityFreeEmail.tsx

```diff
@@ -110,8 +110,10 @@ export const MonthlyActivityFreeEmail = (
       ? "email-monthly-report-free-summary-auto-removed"
       : "email-monthly-report-free-broker-scan-available",
     // Show number of free scans if a scan hasn't been run
-    // If a free scan is run, show 0 auto-removed exposures
-    dataPointValue: hasRunFreeScan ? 0 : 1,
+    // If a free scan is run, show auto-removed exposures (should be 0, unless a user has an expired sub with previously removed exposures)
+    dataPointValue: hasRunFreeScan
+      ? props.dataSummary.dataBrokerAutoFixedDataPointsNum
+      : 1,
```

Comment:
Question: Is showing the auto-removed exposures as `1` when no scan has been run correct here: Wouldn’t there be no exposures that could have been removed?